templates/seL4RPCDataport-to.template.h

```diff
@@ -3,7 +3,6 @@
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
-  #*/
```

Comment:
This one is not a preprocessor directive, it's a template comment that has gone wrong (probably may fault). It should be

```
/*#
 * ..
#*/
```

Reply:
It was already fixed in https://github.com/seL4/global-components/commit/02133497ed126910ec0f4ddb98b997e481d91ffa I believe

Reply:
Ah, yes, hence the conflicts.